src/Tizen.NUI/src/internal/Common/PropertyHelper.cs

```diff
@@ -170,13 +170,11 @@ private static PropertyValue PropertyValueColorToVector4(PropertyValue value)
                 return null;
             }
 
-            var colorValue = new Vector4();
-            value.Get(colorValue);
-            using (var v4 = new Vector4(colorValue.R, colorValue.G, colorValue.B, colorValue.A))
+            return ReusablePool<Vector4>.GetOne((vector4, value) =>
```

Comment:
@hinohie What is the original intent of this code?

Wouldn't it be better to use the copy constructor of PropertyValue? What do you think?

```suggestion
            return new PropertyValue(value);
```

Reply:
When I check the history, The `PropertyValueColorToVector4` was exist as `PropertyValueColorToVector3` for MIX_COLOR and `PropertyValueColorToAlpha` for OPACITY. 

But after dali engine change MIX_COLOR as Vector4 type, we don't need to convert it anymore. (Relative PR : #6413)

So, my opinion is, `PropertyValueColorToVector4` and it's relative codes are legacy now. They be removed.

But, this job looks another phase of PR. So for now, just use copy constructor is enough.
I'll remove them after codes are stable